packages/odyssey-react-mui/src/properties/translations/odyssey-react-mui_zh_TW.properties

```diff
@@ -14,5 +14,22 @@ table.fetchedrows.text = \u5DF2\u64F7\u53D6 {{totalRows}} \u5217
 table.fetchedrows.text_plural = \u5DF2\u64F7\u53D6\u7E3D\u5171 {{totalRows}} \u5217
 table.rows.text = {{totalRows}} \u5217
 table.rows.text_plural = {{totalRows}} \u5217
+table.actions = [[odyssey:odyssey-react-mui: table.actions]]
+table.draghandle.tooltip = [[odyssey:odyssey-react-mui: table.draghandle.tooltip]]
+table.draghandle.arialabel = [[odyssey:odyssey-react-mui: table.draghandle.arialabel]]
+table.reorder.tofront = [[odyssey:odyssey-react-mui: table.reorder.tofront]]
+table.reorder.forward = [[odyssey:odyssey-react-mui: table.reorder.forward]]
+table.reorder.backward = [[odyssey:odyssey-react-mui: table.reorder.backward]]
+table.reorder.toback = [[odyssey:odyssey-react-mui: table.reorder.toback]]
+table.density.arialabel = [[odyssey:odyssey-react-mui: table.density.arialabel]]
+table.columnvisibility.arialabel = [[odyssey:odyssey-react-mui: table.columnvisibility.arialabel]]
+table.moreactions.arialabel = [[odyssey:odyssey-react-mui: table.moreactions.arialabel]]
+filters.filters.arialabel = [[odyssey:odyssey-react-mui: filters.filters.arialabel]]
+filters.filter.any = [[odyssey:odyssey-react-mui: filters.filter.any]]
+filters.filter.clear = [[odyssey:odyssey-react-mui: filters.filter.clear]]
+filters.menuitem.selected = [[odyssey:odyssey-react-mui: filters.menuitem.selected]]
+filters.menuitem.any = [[odyssey:odyssey-react-mui: filters.menuitem.any]]
+filters.clear.label = [[odyssey:odyssey-react-mui: filters.clear.label]]
+filters.search.label = [[odyssey:odyssey-react-mui: filters.search.label]]
```

Comment:
Instead of all this, did you try that `Partial<typeof en>` fix? It's a temp fix until I figure it out.